plugins/outputs/newrelic/README.md

```diff
@@ -0,0 +1,17 @@
+#New Relic output plugin
+  
+This plugins writes to New Relic insights.
+
+```
+[[outputs.newrelic]]
+## New Relic Insights API key
+insights_key = "insights api key"
+##metric_prefix if defined, prefix's metrics name for easy identification
+# metric_prefix = "Telegraf_"
```

Comment:
@hsinghkalsi I'd recommend making this "" by default.  At least that's what we decided for the use case we are trialing this form.